cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -105,7 +106,7 @@ def engine(self) -> 'engine_base.Engine':
     @util.deprecated_gate_set_parameter
     def get_sampler(
         self, gate_set: Optional[serializer.Serializer] = None
-    ) -> engine_sampler.QuantumEngineSampler:
+    ) -> 'cg.engine.QuantumProcessorSampler':
```

Comment:
@wcourtney Will changing this cause any backwards incompatibility problems?   This affects the QCS engine in addition to the simulated version.  

I think it should be fine, but want to double-check.

Reply:
This is a backwards incompatible change for anyone using `processor.get_sampler()` and trying to use multiple processor_ids. However, this workflow doesn't make any sense and I don't think many people are using `processor.get_sampler()`.

Most people using `cg.get_engine().get_sampler()` or `cg.get_engine_sampler()` will be unaffected

Reply:
I updated the PR description